### DISCUSSION TITLE
Retrieve Reactions (Emojis) on Messages

### DISCUSSION THREAD

**Discussion by johe123qwe** [Terrance/SkPy#272] [General]
Hi everyone,

I'm currently using skpy for sending and receiving messages via Skype, and I wanted to ask if there is any existing solution or workaround to retrieve the reactions (such as emojis) that people give to my messages. Specifically, I'd like to collect information about which users reacted to a message and what reaction (emoji) they used.

I understand that this might not be supported natively in skpy at the moment, but I was wondering if anyone has successfully managed to access this information through skpy or other methods, such as interacting with Skype's Web API or any other approach.

If not, is this a feature that could potentially be added in future versions of skpy?

**Comment by Terrance** [comment]
Feature tracked in #56 -- pull requests welcome to expose this as a property of `SkypeMsg`.

I would expect the API to not have changed in the meantime, so this information can probably be found inside `.raw`.
